src/components/HypothesisTesting/PerformTest.js

```diff
@@ -9,45 +9,111 @@ import { jStat } from "jstat";
 import DataDisplay from "./DataDisplay.js";
 import SampleSizeAlphaInputs from "./SampleSizeAlphaInput.js";
 import { popShapeType } from "../../lib/types.js";
+import {InputGroupText, Input } from "reactstrap";
+
+export default function PerformTest({ testType, distType, shape, tails, mue0 }) {
 
-export default function PerformTest({ distType, shape, tails, mue0 }) {
   const [popArr, setPopArr] = useState([]);
   const [sample, setSample] = useState([]);
   const [sampleSize, setSampleSize] = useState(0);
   const [alpha, setAlpha] = useState(0);
   const [sim, setSim] = useState(0);
 
+  const [popArr2, setPopArr2] = useState([]);
+  const [sample2, setSample2] = useState([]);
+  const [sampleSize2, setSampleSize2] = useState(0);
+  const [sim2, setSim2] = useState(0);
+
+  let popMean2 = Math.random(61,66);
```

Comment:
You should probably just move this variable into the `useEffect`, as it is not used anywhere else, then remove it from the dependency array. Also you should probably make it a `const`.